### ISSUE TITLE
Malicious site is not detected

### ISSUE THREAD

**Issue by Heavenwalker** [afilipovich/gglsbl#15]
I was testing the implementation and while it works fine for most of the urls i checked there is one in particular that passes every time while it's being blocked by the lookup API (tested it on Chrome). All urls were checked with gglsbl_client.py but for this particular one, not even a hash is being created. I thought you might wanna take a look as it might be a broader issue.

here is the malicious Url:
http://www.paypal.com.verifyaccount-update.com/us/webapps/m812/home

**Comment by bAndie91** [comment]
it's not blacklisted at me too.
my last gglsbl update log:
```
2017-02-01 11:11:28,470 DEBUG Storing sub chunk #253817 for list name goog-malware-shavar
2017-02-01 11:11:28,491 DEBUG Storing sub chunk #253818 for list name goog-malware-shavar
2017-02-01 11:11:28,534 DEBUG Storing add chunk #258513 for list name goog-malware-shavar
2017-02-01 11:11:28,646 DEBUG Storing sub chunk #123093 for list name googpub-phish-shavar
2017-02-01 11:11:28,659 DEBUG Storing add chunk #426011 for list name googpub-phish-shavar
2017-02-01 11:11:28,993 DEBUG Storing sub chunk #83463 for list name goog-unwanted-shavar
2017-02-01 11:11:28,999 DEBUG Storing add chunk #86680 for list name goog-unwanted-shavar
```

**Comment by Heavenwalker** [comment]
But still Chrome,Firefox and Opera (which , i believe, does not use Safebrowsing ) raise a deceptive site warning

**Comment by afilipovich** [comment]
@Heavenwalker: thank you for reporting the issue. Looks like that URL is blacklisted in the list "SOCIAL_ENGINEERING/OSX/URL" which is not checked by default as I assumed it is a subset of "SOCIAL_ENGINEERING/ANY_PLATFORM/URL". I am going to update the library to check all available blacklists.

@bAndie91: from the log attached it looks like you are using older version of gglsbl which relies on Safe Browsing API v3 which is scheduled for deprecation. You may want to update to the lastest version of gglsbl to take advantage of API v4 (it is a bit quicker).

**Comment by Heavenwalker** [comment]
Thank you very much. Please inform me when you implement the fix so i can test again.

**Comment by afilipovich** [comment]
Pushed the fix to master branch.

**Comment by Heavenwalker** [comment]
Seems to be working perfectly now. Thank you very much.

**Comment by afilipovich** [comment]
Thanks for confirming, I am going to tag a release for it.

**Comment by Heavenwalker** [comment]
Hello it's me again with the same issue ( don't know if i'm supposed to open a new issue). This time it's this link 

http://bc.vc/onaVNqY

**Comment by Heavenwalker** [comment]
Interestingly enough, Safebrowsing's transparency report report the particular url as safe... But if you try to access it with Google Chrome you get the classic "Deceptive Site Ahead Warning"

**Comment by afilipovich** [comment]
I just ran the test with updated cache and this URL gets reported as blacklisted:
`http://babycot.net.au/wp-includes/js/jquery/Admin/pdffolder/pdf.html is blacklisted in [SOCIAL_ENGINEERING/OSX/URL, SOCIAL_ENGINEERING/LINUX/URL, SOCIAL_ENGINEERING/ANY_PLATFORM/URL, SOCIAL_ENGINEERING/WINDOWS/URL, SOCIAL_ENGINEERING/CHROME/URL, SOCIAL_ENGINEERING/ALL_PLATFORMS/URL]
`
